lib/rubocop/cop/lint/missing_cop_enable_directive.rb

```diff
@@ -1,6 +1,5 @@
 # frozen_string_literal: true
 
-# rubocop:disable Lint/RedundantCopDisableDirective
```

Comment:
I believe it was a bug in old `CommentConfig`: It treated code like this:
```ruby
# Example of usage:
#  ```ruby
#  # rubocop:disable SomeCop
```
...as a real directive. I fixed it, but making sure that `rubocop:<directive_name>` is the first thing in comment.
Therefore some of disable/enable throughout codebase, made to workaround this problem, became irrelevant and were removed.